app/models/concerns/searcheable.rb

```diff
@@ -42,7 +42,7 @@ def search_with_params(params)
         .then { |scope| params[:year].present? ? scope.by_year(params[:year]) : scope }
         .then { |scope| params[:month].present? ? scope.by_month(params[:month]) : scope }
         .then { |scope| params[:query].present? ? scope.search(params[:query]) : scope }
-        .then { |scope| scope.order(created_at: sort_order(params[:order])) }
+        .then { |scope| scope.order(created_at: sort_order(params[:order]&.to_sym)) }
```

Comment:
```suggestion
        .then { |scope| params[:order].present? ? scope.order(created_at: sort_order(params[:order].to_sym)) : scope }
```

Reply:
I guess when we don't call any method from concern it return model class itself instead of relation class.
Changing to scope should be ok.